lib/_tls_wrap.js

```diff
@@ -881,6 +889,8 @@ exports.connect = function(/* [port, host], options, cb */) {
     options.singleUse = true;
 
   assert(typeof options.checkServerIdentity === 'function');
+  assert(typeof options.minDHSize === 'number');
+  assert(options.minDHSize > 0);
```

Comment:
Can you add assert messages?  I'd update the options.checkServerIdentity assert as well while you're here.